ui/components/app/signature-request-original/__snapshots__/signature-request-original.test.js.snap

```diff
@@ -167,7 +167,7 @@ exports[`SignatureRequestOriginal should match snapshot 1`] = `
           <h6
             class="box box--margin-bottom-1 box--flex-direction-row typography typography--h6 typography--weight-bold typography--style-normal typography--align-end typography--color-text-default"
           >
-            0
+            966.987986
```

Comment:
This change is expected due to a mockdata change and is to validate the balance inSign typed data v3 feature